### PR TITLE
Modified Schema

### PR DESCRIPTION
**Pull request by pragya-joshi**

### CODE REVIEW

File: server/models/schema.js [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
-    noOfOpenings:Number,
-    noOfSeekers:Number,
+    numberOfOpenings:Number,
+    // seekersRegistered:[{ type: mongoose.Schema.Types.SeekerId, ref: 'Seekers' }],
     shortlistedCount:Number
 });
 
 const Jobs = mongoose.model("Jobs",JobSchema);
 
+const job1=new Jobs({
```
Comment (ankitmishra03) [inline server/models/schema.js:26]: remove

File: server/models/schema.js [hunk 2]
```diff
@@ -1,30 +1,49 @@
...
+    jobId:"j1101",
+    companyName:"Amazon",
+    jobDate:"2023-01-10",
+    jobLocation:"Mumbai",
+    isExpired:false,
+    numberOfOfOpenings:10
+})
+
+job1.save();
```
Comment (ankitmishra03) [inline server/models/schema.js:35]: remove

File: server/models/schema.js [hunk 3]
```diff
@@ -36,23 +55,50 @@ const JobSeekerSchema = new mongoose.Schema({
...
     contactNumber:{
         type: Number,
         required: false
     },
-    jobsApplied:[{ type: mongoose.Schema.Types.ObjectId, ref: 'Jobs' }]
+    appliedJobList:[{
+        jobid: {
+            type: String, ref: 'Jobs'
+        },
+        referralStatus:Boolean
```
Comment (ankitmishra03) [inline server/models/schema.js:70]: make it default:false

File: server/models/schema.js [hunk 4]
```diff
@@ -36,23 +55,50 @@ const JobSeekerSchema = new mongoose.Schema({
...
 
+const Seekers = mongoose.model("Seekers",JobSeekerSchema);
 
+const appliedJobInfo={
+    jobid:job1.jobId,
+    referralStatus:false
+}
+
+
+const seeker1=new Seekers({
```
Comment (ankitmishra03) [inline server/models/schema.js:83]: remove

File: server/models/schema.js [hunk 5]
```diff
@@ -36,23 +55,50 @@ const JobSeekerSchema = new mongoose.Schema({
...
+    seekerEmail:"pragya@gmail.com",
+    password:"1234",
+    collegeName:"STCET",
+    contactNumber:8564237154,
+    appliedJobList:[appliedJobInfo]  //storing the job id's of the jobs the seeker has applied to
+})
 
-const Seekers = mongoose.model("Seekers",JobSeekerSchema);
+
+seeker1.save();
```
Comment (ankitmishra03) [inline server/models/schema.js:94]: remove

File: server/models/schema.js [hunk 6]
```diff
@@ -64,16 +110,17 @@ const EmployeeSchema = new mongoose.Schema({
...
         type: String,
         required: true
     },
-    // referralStatus:String,
     contactNumber:{
         type: Number,
         required: false
     },
-    newJobPosts:[JobSchema]
+    listOfJobsPosted:[{ type: mongoose.Schema.Types.ObjectId, ref: 'Jobs' }],
```
Comment (ankitmishra03) [inline server/models/schema.js:121]: make it same as appliedJobList

File: server/models/schema.js [hunk 7]
```diff
@@ -64,16 +110,17 @@ const EmployeeSchema = new mongoose.Schema({
         type: String,
         required: false
     },
-    companyName:{
+    employeeCompany:{
```
Comment (bhavneet2000) [inline server/models/schema.js:113]: change to employeeCompanyName, to make it consistent with seeker.

File: server/models/schema.js [hunk 8]
```diff
@@ -1,30 +1,49 @@
...
+    jobDate:"2023-01-10",
+    jobLocation:"Mumbai",
+    isExpired:false,
+    numberOfOfOpenings:10
+})
+
+job1.save();
+
 const JobSeekerSchema = new mongoose.Schema({
-    name: {
```
Comment (bhavneet2000) [inline server/models/schema.js:23]: Add a key for resumeUrl, which would be string and mandatory for all seekers.

**Review by ankitmishra03 — APPROVED** [review]

**Review by ankitmishra03 — APPROVED** [review]

**Pull request merged**